maintenance/enableDiscussionsWithNoForums.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Enables Discussions on the given wikis, after verifying there are no threaded forum posts on each wiki
+ * Usage: php enableDiscussionsWithNoForums.php <listfile>
+ * Where:
+ * 	<listfile> is a file where each line is a site ID where we want to enable discussions
+ *
+ * @ingroup Maintenance
+ */
+use Swagger\Client\ApiException;
+use Swagger\Client\Discussion\Api\SitesApi;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Swagger\ApiProvider;
+require_once( __DIR__ . '/Maintenance.php' );
+
+class EnableDiscussionsWithNoForums extends Maintenance {
+
+	const CURL_TIMEOUT = 5;
+	const SERVICE_NAME = 'discussion';
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Enables Discussions on given wikis with no threaded forum posts";
+		$this->addArg( 'listfile', 'File with site IDs to enable discussions on, separated by newlines.' );
+	}
+
+	public function execute() {
+		$fh = fopen( $this->getArg() , 'r' );
+
+		if ( !$fh ) {
+			$this->error( 'Unable to read from input file, exiting', true );
+		}
+
+		$schwartzToken = F::app()->wg->TheSchwartzSecretToken;
+
+		while ( !empty( $wikiId = trim( fgets( $fh ) ) ) ) {
+			$wiki =  WikiFactory::getWikiByID( $wikiId );
+			$dbw = wfGetDB( DB_SLAVE, [], $wiki->city_dbname );
+			$row = $dbw->selectRow(
+				[ 'comments_index', 'page' ],
+				[ 'count(*) cnt' ],
+				[
+					'parent_comment_id' => 0,
+					'archived' => 0,
+					'deleted' => 0,
+					'removed' => 0,
+					'page_namespace' => NS_WIKIA_FORUM_BOARD_THREAD
+				],
+				__METHOD__,
+				[ ],
+				[ 'page' => [ 'LEFT JOIN', [ 'page_id=comment_id' ] ] ]
+			);
+
+			$totalThreads = intval( $row->cnt );
```

Comment:
Line 38 to line 54 look like a good candidate for a `getForumThreadCount()` method!